src/components/POLine/POLineInvoices/POLineInvoicesContainer.js

```diff
@@ -43,6 +55,7 @@ POLineInvoicesContainer.propTypes = {
   // eslint-disable-next-line react/no-unused-prop-types
   lineId: PropTypes.string.isRequired,
   label: PropTypes.object.isRequired,
+  stripes: PropTypes.object.isRequired,
   accordionId: PropTypes.string.isRequired,
```

Comment:
Guys, maybe I'm missing something, but what is the reason of this prop, `accordionId` ? It looks to me that just a constant is passed down

Reply:
You are right